engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobStarterActor.scala

```diff
@@ -0,0 +1,122 @@
+package cromwell.engine.workflow.lifecycle.execution
+
+import akka.actor.{LoggingFSM, Props}
+import cromwell.backend._
+import cromwell.engine.EngineWorkflowDescriptor
+import cromwell.engine.workflow.lifecycle.execution.JobStarterActor._
+import wdl4s._
+import wdl4s.expression.WdlStandardLibraryFunctions
+import wdl4s.util.TryUtil
+import wdl4s.values.WdlValue
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+
+object JobStarterActor {
+  /** Commands */
+  case object Start
+  /** Responses */
+  case class BackendJobStartSucceeded(jobDescriptor: BackendJobDescriptor, props: Props)
+  case class BackendJobStartFailed(jobKey: JobKey, throwable: Throwable)
+  /** Data */
+  class JobStarterActorStateData
+  /** States */
+  sealed trait JobStarterActorState
+  case object Idle extends JobStarterActorState
+  case object EvaluatingInputs extends JobStarterActorState
+  /** Internal Messages*/
+  private case class EvaluatedInputs(inputs: Map[LocallyQualifiedName, WdlValue])
+
+  def props(executionData: WorkflowExecutionActorData,
+            jobKey: BackendJobDescriptorKey,
+            factory: BackendLifecycleActorFactory,
+            configDescriptor: BackendConfigurationDescriptor) = Props(new JobStarterActor(executionData, jobKey, factory, configDescriptor))
+}
+
+case class JobStarterActor(executionData: WorkflowExecutionActorData,
+                           jobKey: BackendJobDescriptorKey,
+                           factory: BackendLifecycleActorFactory,
+                           configDescriptor: BackendConfigurationDescriptor) extends LoggingFSM[JobStarterActorState, JobStarterActorStateData] with WdlLookup {
+
+  override val workflowDescriptor: EngineWorkflowDescriptor = executionData.workflowDescriptor
+  override val executionStore: ExecutionStore = executionData.executionStore
+  override val outputStore: OutputStore = executionData.outputStore
+  override val expressionLanguageFunctions = executionData.expressionLanguageFunctions
+
+  startWith(Idle, new JobStarterActorStateData())
```

Comment:
The StateData can probably just be `Unit` if we don't intend on using `JobStarterActorStateData`